eras/byron/ledger/executable-spec/byron-spec-ledger.cabal

```diff
@@ -73,16 +73,16 @@ test-suite byron-spec-ledger-test
         "-with-rtsopts=-K4m -M150m"
 
     build-depends:
-        base,
-        bimap,
+        base >=4.14 && <4.17,
```

Comment:
I removed those constraints on purpose, because there is no point in duplicating the same constraints for the library and the test-suite at the same time